katas/content/linear_algebra/index.md

```diff
@@ -392,33 +392,32 @@ $$A = \begin{bmatrix}
 
 <details>
 <summary><b>Solution</b></summary>
-To check whether the input matrix is unitary, we will need to perform the following steps:
+To check whether the input matrix is unitary, you need to perform the following steps:
 
 1. Calculate the adjoint of the input matrix $A^\dagger$.
 
-$$A^\dagger = \frac{1}{\sqrt{2}} \begin{bmatrix}
-    1 & -i \\
-    1 & i
-\end{bmatrix}$$
-
-2. Multiply it by the input matrix.
-
-$$AA^\dagger = \frac12 \begin{bmatrix}
-    1 & 1 \\
-    i & -i
-\end{bmatrix} \begin{bmatrix}
-    1 & -i \\
-    1 & i
-\end{bmatrix} = \frac12 \begin{bmatrix}
-    1 \cdot 1 + 1 \cdot 1 & 1 \cdot (-i) + 1 \cdot i \\
-    i \cdot 1 + (-i) \cdot 1 & i \cdot (-i) + (-i) \cdot i
-\end{bmatrix} = \begin{bmatrix}
-    1 & 0 \\
-    0 & 1
-\end{bmatrix}$$
-
-If the multiplication result $AA^\dagger$ is an identity matrix, which is indeed the case, 
-and the product $A^\dagger A$ is also an identity matrix (which you can verify in a similar manner),
+    $$A^\dagger = \frac{1}{\sqrt{2}} \begin{bmatrix}
+        1 & -i \\
+        1 & i
+    \end{bmatrix}$$
+
+1. Multiply it by the input matrix.
```

Comment:
Please use distinct numbers for Markdown lists, using the same number makes readability worse even if the parser accepts it
```suggestion
2. Multiply it by the input matrix.
```